### DISCUSSION TITLE
apollo集成consul 多环境出现yml配置同步问题

### DISCUSSION THREAD

**Discussion by nongmingdashen** [apolloconfig/apollo#3828] [Q&A]
服务器2台  分别   214（apollo-config、apollo-admin）   215（apollo-config、apollo-admin、apollo-portal）
环境情况：打包apollo一次  build.sh  
环境配置：
   config-db  214
   portal-db  215

   dev_meta=214:9100
   pro_meta=215:9100

   拷贝apollo-config、apollo-admin 到214     拷贝 apollo-config、apollo-admin、apollo-portal  215服务器
   214直接运行start

   215分别修改apollo-config、apollo-admin下面  config目录application-github.properties文件的数据库连接信息
   然后运行215 apollo-config、apollo-admin、apollo-portal  
    
   修改数据库apolloportalDB中 serverconfig表   key为apollo.portal.envs的value    dev,pro
    
问题描述：
    apollo正常打开，环境正常  dev、pro  都有。   增加私有的namespace后  发布其中任意一个yml内容  另外一个环境也自动修改。如：为dev环境的yml增加   内容   1111     pro环境也跟随增加了1111    dev点击发布  pro也跟随发布了

 期望：
     不同环境，不同配置。

**Comment by nobodyiam** [comment]
consul 看下是不是连得是同一个，214 和 215 是两个不同的环境，需要连接不同的 consul
同时可以看一下 管理员工具 - 系统信息 页面

**Reply by nongmingdashen** [reply]
是连接的不同环境，这个我确认
